### ISSUE TITLE
LUA errors after the last update 

### ISSUE THREAD

**Issue by Nillx** [FabysGames/mythicplustimer#1]
Getting this errors when i enter a timewalking dungeon

```
31x MythicPlusTimer\infos.lua:168: attempt to index local 'current_run' (a nil value)
MythicPlusTimer\infos.lua:168: in function <MythicPlusTimer\infos.lua:166>
MythicPlusTimer\infos.lua:327: in function `update_reaping'
MythicPlusTimer\criteria.lua:281: in function `update'
MythicPlusTimer\criteria.lua:223: in function `callback'
MythicPlusTimer\addon.lua:100: in function <MythicPlusTimer\addon.lua:94>

Locals:
InCombatSkipped
```

```
2x MythicPlusTimer\criteria.lua:71: attempt to index local 'current_run' (a nil value)
MythicPlusTimer\criteria.lua:71: in function <MythicPlusTimer\criteria.lua:69>
MythicPlusTimer\criteria.lua:291: in function `update_step'
MythicPlusTimer\criteria.lua:277: in function `update'
MythicPlusTimer\criteria.lua:223: in function `callback'
MythicPlusTimer\addon.lua:100: in function <MythicPlusTimer\addon.lua:94>

Locals:
InCombatSkipped
```

**Comment by fabysdev** [comment]
Thanks, looking at it

**Comment by FabysGames** [comment]
thanks for testing the beta version.

will look at it

**Comment by FabysGames** [comment]
Should be fixed in 3.0-Beta-2.
